ptf_tests/common/utils/gnmi_cli_utils.py

```diff
@@ -97,7 +98,8 @@ def gnmi_get_params_counter(param):
          port_counter[each] = int(next(iter_rslt))
 
     if port_counter:
-           return port_counter
+        print (f"PASS: The port {param.split(',')[1].split(':')[1]} counter is built successfully")

```

Comment:
can we move this print statement to the test case?

Reply:
update the code accordingly.